src/layer/heatmap/HeatMapLayer.js

```diff
@@ -51,12 +51,18 @@ define([
      * @classdesc A HeatMap layer for visualising an array of measured locations.
      * @param {String} displayName This layer's display name.
      * @param {MeasuredLocation[]} measuredLocations An array of locations with measures to visualise.
+     * @param {Number} amountOfLevels Optional. If provided it specifies the amount of levels that will be generated for
```

Comment:
`TileImageLayer` calls this property `numLevels`. Please use the same name for consistency.